index.html

```diff
@@ -3,16 +3,34 @@
 <head>
     <meta charset="UTF-8">
     <title>phzipsearch</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css">
```

Comment:
Thanks @frozeeen.

But sorry, we don't do CDN in this company. XD

Reply:
To keep it simple, please just download [Bulma](https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css) to `/public/css` and link it from there.